Java/sort/ShellSort.java

```diff
@@ -0,0 +1,30 @@
+import java.util.*;
+
+public class ShellSort{
+    public static void main(String[] args){
+        int array[] = {30, 25, 77, 16 , 80, 32, 67} ;
+        shellSort(array);
+    }
+
+    public static void shellSort(int array[]){
+        int size = array.length;
```

Comment:
Include appropriate Documentation (docstrings) for functions. Add comments where needed.